ur_controllers/include/ur_controllers/gpio_controller.hpp

```diff
@@ -79,6 +80,8 @@ enum CommandInterfaces
   ZERO_FTSENSOR_ASYNC_SUCCESS = 32,
   HAND_BACK_CONTROL_CMD = 33,
   HAND_BACK_CONTROL_ASYNC_SUCCESS = 34,
+  GET_VERSION_CMD = 35,
+  GET_VERSION_ASYNC_SUCCESS = 36
```

Comment:
Thinking about this... I don't think, we actually need command interfaces. The information that we want to publish, doesn't change over the runtime of this controller. We could initially fill the values inside the hardware interface when configuring the hardware and then just fill the service response from those.